.golangci.yml

```diff
@@ -0,0 +1,80 @@
+run:
+  build-tags: 
+    - aws
+    - packet
+    - aks
+    - e2e
+    - baremetal
+    - disruptivee2e
+    - poste2e
+
+  modules-download-mode: vendor
+
+  timeout: 5m
+
+issues:
+  exclude-use-default: false
+  max-same-issues: 0
+  max-issues-per-linter: 0
+
+linters:
+  disable-all: false
+  enable:
+    - asciicheck
+    - bodyclose
+    - deadcode
+    - depguard
+    - dogsled
+    - dupl
+    - errcheck
+    - exhaustive
+    - funlen
+    - gochecknoinits
+    - gocognit
+    - goconst
+    - gocritic
+    - gocyclo
+    - godot
+    - gofmt
+    - gofumpt
+    - goimports
+    - golint
+    - gomnd
+    - goprintffuncname
+    - gosec
+    - gosimple
+    - govet
+    - ineffassign
+    - interfacer
+    - lll
+    - maligned
+    - misspell
+    - nakedret
+    - nestif
+    - noctx
+    - nolintlint
+    - prealloc
+    - rowserrcheck
+    - scopelint
+    - staticcheck
+    - structcheck
+    - stylecheck
+    - testpackage
+    - typecheck
+    - unconvert
+    - unparam
+    - unused
+    - varcheck
+    - whitespace
+    - wsl
+    # - exportloopref
```

Comment:
What's up with those? Why they are commented out?

Reply:
Oh man, I had mixed mind regarding adding then to disabled list or commenting them, after reading your comment 🤦. Fixing it